test/thermodynamics/average_square_of_velocity_test.py

```diff
@@ -0,0 +1,46 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    Quantity,
+    SI,
+    convert_to,
+)
+from symplyphysics.laws.thermodynamics import average_square_of_velocity as velocity_law
+
+# Description
+## Example from https://easyfizika.ru/zadachi/molekulyarnaya-fizika/opredelit-srednyuyu-kvadratichnuyu-skorost-molekul-vodoroda/
+## The ideal gas has a temperature of 273 K. The mass of one atom is equal to 3,32 * 10^(-27) kg. Then the average square of velocity should be 3.403 * 10^6 (m/s)^2
```

Comment:
It is not mass of an atom of hydrogen.

Reply:
If molar mass of molecule $H_2$? 
$\mu(H_2) = 2 * \mu(H) = 2 * 0.001 = 0.002 (\text{kg/mol})$
Then
$m = \mu(H_2) / N_A = 0.002 / (6.022 * 10^{23}) = 3.32 * 10^{-27} (\text{kg})$

Reply:
Yes. It's mass of molecule. Molecule has 2 atoms of hydrogen.

Reply:
Thank you for your patience and understanding!